src/app/rundown-view/components/rundown-tooltips/video-hover-scrub/video-hover-scrub.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, ElementRef, Input, OnDestroy, OnInit, ViewChild } from '@angular/core'
+import { Tv2PieceType } from 'src/app/core/enums/tv2-piece-type'
+import { PieceLifespan } from 'src/app/core/models/piece-lifespan'
+import { Observable, Subject, takeUntil } from 'rxjs'
+import { ConfigurationService } from 'src/app/shared/services/configuration.service'
+import { StudioConfiguration } from 'src/app/shared/models/studio-configuration'
+import { IconButton, IconButtonSize } from 'src/app/shared/enums/icon-button'
+
+const ROUNDING_PRECISION: number = 100
+
+@Component({
+  selector: 'sofie-video-hover-scrub',
+  templateUrl: './video-hover-scrub.component.html',
+  styleUrls: ['./video-hover-scrub.component.scss'],
+})
+export class VideoHoverScrubComponent implements OnInit, OnDestroy {
+  @Input() public fileName: string
```

Comment:
Filename is with a lowercased `n`

Reply:
I think we have a lot of inconsistencies on this attribute (and I think I introduced them...) we should decide on which format we want and clean it up.